### ISSUE TITLE
Loading XML file in custom themes fails for selection background color

### ISSUE THREAD

**Issue by philipyun** [ChrisRM/material-theme-jetbrains#1301]
### Summary

XML Files being manually loaded load the incorrect color for Selection Background

...

### Expected behavior

Load the correct color from the XML file for "selectionBackground"
...

### Actual behavior
Loads the background Color instead
...

### Steps to reproduce (if needed)

1. Go To Custom theme
2. Load XML
3. use following xml

```
<mtTheme>
  <id>test</id>
  <editorColorsScheme>test</editorColorsScheme>
  <dark>true</dark>
  <name>test</name>
  <colors>
    <color id="background" value="272822ff"/>
    <color id="foreground" value="f8f8f2ff"/>
    <color id="text" value="f8f8f2ff"/>
    <color id="highlight" value="3f3f3fff"/>
    <color id="selectionBackground" value="d20f00ff"/>
    <color id="selectionForeground" value="f8f8f2ff"/>
    <color id="button" value="2a2a2aff"/>
    <color id="secondaryBackground" value="272822ff"/>
    <color id="disabled" value="474747ff"/>
    <color id="contrast" value="1e1f1cff"/>
    <color id="tableSelected" value="4a4c41ff"/>
    <color id="secondBorder" value="4a4c41ff"/>
    <color id="treeSelection" value="4a4c41c0"/>
    <color id="notifications" value="1a1a1aff"/>
    <color id="accent" value="a39f8a7e"/>
    <color id="excluded" value="323232ff"/>
  </colors>
  <themeId>test</themeId>
</mtTheme>
```

You can also add Screenshots for a better understanding.

### Environment

WebStorm 2019.2
Build #WS-192.5728.87, built on July 22, 2019
Runtime version: 11.0.3+12-b304.10 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.6
GC: ParNew, ConcurrentMarkSweep
Memory: 1981M
Cores: 8

Registry: ide.mac.allowDarkWindowDecorations=true, ide.tree.ui.experimental=false, ide.balloon.shadow.size=0
Non-Bundled Plugins: br.com.matheusfm.jsonutils, color.scheme.Monokai, color.scheme.Monokai Pro Spectrum, com.chrisrm.idea.MaterialThemeUI, com.intellij.plugins.vscodekeymap, com.deadlock.scsyntax, org.sylfra.idea.plugins.linessorter

Plugin version 4.2.1

**Comment by mallowigi** [comment]
Maybe it’s because it’s a 8 digits color, try to add opacity at the end, like ff for fully opaque

**Comment by philipyun** [comment]
@mallowigi Sorry, the color code was 6 digits, cus i manually edited it to test it out.  Same behavior with the 8 digit code.

I am however realizing that the selectedBackground color instead loads secondaryBackground

**Comment by philipyun** [comment]
potential area of concern (java is not my strong suit but this still looks concerning):

https://github.com/ChrisRM/material-theme-jetbrains/blob/c1dd4f081e052bf3b7bc89f98cfc5520ab48df11/src/main/java/com/chrisrm/idea/config/ui/MTCustomThemeForm.java#L87

**Comment by philipyun** [comment]
I think i've found the bug:

https://github.com/ChrisRM/material-theme-jetbrains/blob/286315dc451a63b209d3d579a595f8ab83aec01c/src/main/java/com/chrisrm/idea/themes/themes/MTAbstractTheme.java#L279-L283

should be 
```
  @NotNull
  @Override
  public final Color getSelectionBackgroundColor() {
    return getSelectionBackgroundResource(); // NOT getSecondaryBackgroundColorResource();
  }
```

**Comment by mallowigi** [comment]
Thanks, I’ll fix it soon

**Comment by mallowigi** [comment]
Fixed in 4.2.2
